### DISCUSSION TITLE
There's that AI again.

### DISCUSSION THREAD

**Discussion by LukasKanka** [helix-editor/helix#10131] [Ideas] [answered]
There have been plenty of topics on AI here already, but I'm asking anyway. I replaced Copilot with Tabnine. And now I've replaced Tabnine with Codeium. It's a great AI for individuals for free, o versus Tabnine free, but it really works. 
Would there be a chance that Helix could ever support Codeium?
https://codeium.com

**Comment by kirawi** [comment, marked as answer]
See https://github.com/helix-editor/helix/discussions/4037
Use https://github.com/leona/helix-gpt

**Reply by LukasKanka** [reply]
Thank You!

**Reply by TornaxO7** [reply]
@LukasKanka please don't forget to mark kirawis message as your answer to your question if it solved your issue :)

**Comment by rosarp** [comment]
@LukasKanka  I have created rust version which configures & connects to codeium ai for completions within helix. 
Configuration is provided  https://github.com/rosarp/llm-lsp/

**Comment by oderwat** [comment]
There is also https://github.com/SilasMarvin/lsp-ai (I did not evaluate it yet)
